backend/routes.py

```
@@ -35,6 +49,194 @@ def register_event_handlers():
+        repo_url = repo_request.repo_url.strip()
+        
+        # Handle SSH format (git@github.com:owner/repo.git)
+        if repo_url.startswith("git@"):
+            # Split on ':' to get the path part
+            if ":" in repo_url:
+                path = repo_url.split(":", 1)[1]
+            else:
+                raise HTTPException(
+                    status_code=400,
+                    detail="Invalid GitHub repository URL format"
+                )
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)